ui/src/components/CopyToClipboardTableCell/index.jsx

```diff
@@ -1,15 +1,23 @@
 import React from 'react';
 import { node, string } from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
 import { CopyToClipboard } from 'react-copy-to-clipboard';
 import TableCell from '@material-ui/core/TableCell';
 import ContentCopyIcon from 'mdi-react/ContentCopyIcon';
 import CheckIcon from 'mdi-react/CheckIcon';
 import TableCellItem from '../TableCellItem';
 import useCopyToClipboard from '../../utils/useCopyToClipboard';
 
+const styles = theme => ({
+  icon: {
+    marginLeft: theme.spacing(1),
+    marginTop: theme.spacing(1),
+  },
```

Comment:
Using margin top to center things feels rather hacky and may not always work. Would the following work:

```diff
diff --git a/ui/src/components/CopyToClipboardTableCell/index.jsx b/ui/src/components/CopyToClipboardTableCell/index.jsx
index 8daa02c74..a676368ba 100644
--- a/ui/src/components/CopyToClipboardTableCell/index.jsx
+++ b/ui/src/components/CopyToClipboardTableCell/index.jsx
@@ -11,7 +11,7 @@ import useCopyToClipboard from '../../utils/useCopyToClipboard';
 const styles = theme => ({
   icon: {
     marginLeft: theme.spacing(1),
-    marginTop: theme.spacing(1),
+    display: 'flex',
   },
 });
```

Reply:
Yes, you are right.
Resolved